sdk/src/lib.rs

```diff
@@ -82,6 +82,7 @@
 //! # }
 //! ```
 
+pub use asn1::rfc3161::{MessageImprint, TimeStampReq};
```

Comment:
Why did this need to be exposed?

Reply:
Right now the default implementation of send_time_stamp_request in AsyncSigner trait does the required marshalling for the timestamp body (instead of just making the HTTP call). If I need to do the same outside in an implementation (e.g a fetch call in JavaScript) I will have to redo all the ASN.1 serialization of the timestamp request. I was trying to reuse the same by exposing those helpers. If we could refactor the AsyncSigner trait to separate the HTTP request part and the timestmap request marshalling part then no need to expose the same.

Reply:
What do you mean default implementation for AsyncSigner.   We don't have such a thing except for unit tests of the async interfaces.  The AsyncSigner trait does not have references to TimeStampReq or MessageImprint.  Those are implementation detail of those test signers.  If you were to implement your own AsyncSigner outside of the SDK you can do anything you want as long as you meet the trait requirements.

Reply:
I mean the default implementation of the function send_timestamp_request in AsyncSigner trait.  The default_rfc3161_request implementation uses those. I exposed the same to use outside the crate without having to re-implement it.
```rust
    async fn send_timestamp_request(&self, message: &[u8]) -> Option<Result<Vec<u8>>> {
        let headers: Option<Vec<(String, String)>> = self.timestamp_request_headers();

        self.time_authority_url()
            .map(|url| crate::time_stamp::default_rfc3161_request(&url, headers, message))
    }
```

Reply:
I see, there was a change to the Signer API to require time stamping.  Let me do a quick check on if we want to keep this behavior.

Reply:
We do no plan to re-export internal structures.  I am not opposed to a creating a function that returns the TimeStampRequest bytes that you can use to call the TSA url.

Reply:
Sure. I can make that change.

Reply:
Modified the AsyncSigner interface to have a separate method for the message body.  and have a default implementation. Refactor timestamp.rs to accommodate the same.